src/Core/src/ImageSources/ImageSourceServiceConfigurationExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.Maui
+{
+	public static class ImageSourceServiceConfigurationExtensions
+	{
+		public const string ImageDirectoryKey = "ImageDirectory";
+
+		public static void SetImageDirectory(this IImageSourceServiceConfiguration config, string directory) =>
+			config[ImageDirectoryKey] = directory;
```

Comment:
Just an extension method over the config type to make things easier to use. Probably needs to be an IConfiguration concept tbh.